app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) < 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format. Use"
+                         " 'mv source_path destination_path'.")
+
+    source_path = parts[1]
+    destination_path = parts[2]
```

Comment:
Consider using unpacking to assign values from the list to individual variables, as suggested in the checklist: `source_path, destination_path = parts[1], parts[2]`.